west.yml

```diff
@@ -104,7 +104,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: efc9575b9f759ac54f49ab395234eb2c4df40582
+      revision: pull/614/head
```

Comment:
@ibaz-nordic Please update manifest revision.

Reply:
@jfischer-no - CI failure is not related to BLE
https://jenkins-ncs.nordicsemi.no/blue/organizations/jenkins/latest%2Fsub%2Ftest-fw-nrfconnect-nrf-iot_samples/detail/master/395/pipeline